cmd/initializers/dataset/requirements.txt

```diff
@@ -1 +1,3 @@
 huggingface-hub>=0.27.0,<0.28
+pyyaml
+kubernetes
```

Comment:
Please add the kube version, like in SDK: https://github.com/kubeflow/sdk/blob/main/pyproject.toml#L30